notebooks/tel_and_site/subsys_req_ver/tma/SITCOM-1120_az_brake_analysis.ipynb

```diff
@@ -0,0 +1,1197 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.        """
</pre><p>Maybe add a description for output (<code>event</code>) would be helpful. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/120/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<pre class="ql-syntax" spellcheck="false"> The output includes printed DataFrames for user reference and then returns the relevant formatted data based on the user's selection from the provided options.


</pre><p> </p><p>Description:</p><p>The function prints three separate DataFrames containing the following:</p><p>Azimuth Motion State&nbsp;(df_az_state): This DataFrame displays the state of the azimuth motion during the specified time range, including columns for state and timestamp.</p><p> </p><p>Elevation Motion State&nbsp;(df_el_state): Similar to the azimuth motion state, this DataFrame shows the state of the elevation motion, also with state and timestamp columns.</p><p> </p><p>Azimuth System State&nbsp;(df_move): This DataFrame provides information on the motion states</p><p> </p><p>User Prompt:</p><p> </p><p>After displaying the DataFrames, the function prompts the user to input their choice to select an event type. The user can enter either "1", "2", or any other input to choose the corresponding event data.</p><p> </p><p>Formatted Output:</p><p> </p><p>The final output will be the result of the&nbsp;miniformat&nbsp;function applied to the selected DataFrame. </p>